api/python/.pylintrc

```diff
@@ -1,2 +1,5 @@
 [TYPECHECK]
 generated-members=requests.codes.*,responses.*
+
+[pre-commit-hook]
+limit=8.0
```

Comment:
Does this automatically only lint *.py that are staged and dirty?

Reply:
>The git-pylint-commit-hook is a tool for checking Python code using pylint before the code is committed to git.
>
>All files that end with .py or that have python in the she bang will be automatically checked before they are committed to the repo.

https://git-pylint-commit-hook.readthedocs.io/en/latest/index.html